src/hooks/usePrefersColorScheme.ts

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from 'react'
+
+type ColorSchemeType = 'dark' | 'light' | 'no-preference'
+
+const usePrefersColorScheme = (): ColorSchemeType => {
```

Comment:
We should also take the dark theme into account. We have a very similar hook called `useDarkMode`.

Reply:
You mean considering the app's theme to affect the favicon?

Reply:
Yes. The app theme may not match that of the OS.

Reply:
Exactly for that reason, we should only make the favicon contrast with the OS theme. The app theme does not  affect the browser tabs color.

Reply:
I see. Oversight on my behalf.

Does this work nicely with the pairing module/WC? I don't have an iPhone so I can't test it.

Reply:
And the icons are clearly visible in light/dark mode on WC/app?

Reply:
Yes. Those icons have the green circle background (I didn't change it)